app/create/controller.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+module.exports = function ($scope, Postcard) {
+
+  $scope.loading = false;
+  $scope.error = null;
+  $scope.postcard = null;
+
+  $scope.create = function (payload) {
+
+    $scope.payload = { mail_type: 'usps_first_class' };
```

Comment:
You may want to define this outside the scope of `$scope.create`. `views/create.html` refers to `payload` so it should be initialized along with the other scope variables (`loading`, `error`)